integreat_cms/static/src/js/pages/xliff-export-overlay.ts

```diff
@@ -0,0 +1,71 @@
+window.addEventListener("load", () => {
+  addXliffExportOverlayListeners(openXliffExportOverlay);
+  // Set listener for select toggles.
+  document.querySelectorAll(".bulk-select-language").forEach((el) => {
+    el.addEventListener("change", toggleXliffBulkActionButton); 
+  }); 
+  // Set listener for select-all toggle.
+  document.getElementById("select-all-languages")?.addEventListener("change", (toggle) => {
+    document.querySelectorAll(".bulk-select-language").forEach((el) => {
+      (el as HTMLInputElement).checked = (toggle.target as HTMLInputElement).checked; 
+      toggleXliffBulkActionButton(); 
+    });
+  });
+});
+
+
+function addXliffExportOverlayListeners(callback: (overlay: HTMLElement) => void) {
+  const overlay = document.getElementById("xliff_export_overlay");
+  if (overlay) {
+    // Set listener for bulk action execute button.
+    document.getElementById("bulk-action-execute")?.addEventListener("click", (event) => {
+      const xliffOptionIndex = (document.getElementById("xliff-multiple-languages-option") as HTMLOptionElement).index;
+      const selectedIndex = (document.getElementById("bulk-action") as HTMLSelectElement).selectedIndex; 
+      if (xliffOptionIndex === selectedIndex) {
+        event.preventDefault(); 
+        callback(overlay);
```

Comment:
Is there a specific use case why this callback was outsourced? I cannot think of any other action one might want to do here...
```suggestion
        overlay.classList.remove("hidden");
        overlay.classList.add("flex");
```

Reply:
I think, you're right!